target/linux/ramips/dts/mt7621_xiaomi_mir4.dts

```diff
@@ -0,0 +1,187 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "xiaomi,mir4", "mediatek,mt7621-soc";
+	model = "Xiaomi Mi Router 4";
+
+	aliases {
+		led-boot = &led_status_yellow;
+		led-failsafe = &led_status_red;
+		led-running = &led_status_blue;
+		led-upgrade = &led_status_yellow;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_status_red: status_red {
+			label = "mir4:red:status";
+			gpios = <&gpio0 6 GPIO_ACTIVE_LOW>;
+		};
+
+		led_status_blue: status_blue {
+			label = "mir4:blue:status";
+			gpios = <&gpio0 8 GPIO_ACTIVE_LOW>;
+		};
+
+		led_status_yellow: status_yellow {
+			label = "mir4:yellow:status";
+			gpios = <&gpio0 10 GPIO_ACTIVE_LOW>;
+		};
+
+		wan_amber {
```

Comment:
If there is only one color for a LED type, you can remove the color from the node name:
wan_amber -> wan
Same below for lan1/lan2

Reply:
As in this pic ![ports](https://s3.ifanr.com/wp-content/uploads/2018/06/8S5A4611.jpg)

It's green. On MIR3G there are green and amber LEDs, the green one is controlled by the switch and the amber one is controlled by GPIO.

@soikjk Have you tried removing the port LEDs and checking if it still works? If yes, then it's controlled by the switch so you don't need that definition.